package.json

```diff
@@ -34,7 +35,9 @@
     "dep-graph": "nx dep-graph",
     "help": "nx help",
     "storybook:island-ui": "nx run island-ui-storybook:storybook",
-    "ts-node": "node -r ./scripts/ts-node"
+    "ts-node": "ts-node -r tsconfig-paths/register -O '{\"module\":\"commonjs\",\"esModuleInterop\":true}'",
+    "schemas": "test -f libs/api/schema/src/lib/schema.d.ts || echo 'export default () => {}' >> libs/api/schema/src/lib/schema.d.ts && yarn nx run api:contentful-types && yarn affected:schemas",
+    "postinstall": "if [[ ! -z $DOCKER_BUILD ]]; then exit 0; else yarn schemas; fi"
```

Comment:
Minor: This will break if DOCKER_BUILD contains spaces or is an empty string. Consider `if [ ! -z ${DOCKER_BUILD+x} ]`

Reply:
It's common to check for a CI variable. Does that make sense here?

Reply:
The original `CI` variable isn't available inside the docker build. But I wonder if we should name it `CI` for consistency's sake?

Reply:
I renamed it to CI